test/parallel/test-tls-ecdh.js

```diff
@@ -6,6 +6,18 @@ if (!common.hasCrypto) {
   console.log('1..0 # Skipped: missing crypto');
   return;
 }
+/* This test uses only ECDHE-RSA-RC4-SHA, this means:
+ * Key Exchange: ECDH
+ * Encryption: RC4
+ * Bits: 128
+ * The RFC name is TLS_ECDHE_RSA_WITH_RC4_128_SHA
+ * FIPS does NOT support RC4, so we should skip this test. 
+ */
```

Comment:
The cipher can be changed into `ECDHE-RSA-AES128-SHA256` so we need not skip this test.